packages/newspringchurchapp/src/prayer/SavedPrayerList/index.js

```diff
@@ -1,14 +1,73 @@
 import React from 'react';
-import { ScrollView } from 'react-native';
+import { ScrollView, View } from 'react-native';
+import { Query, Mutation } from 'react-apollo';
+import { get } from 'lodash';
+import { styled } from '@apollosproject/ui-kit';
 
 import PrayerCard from '../PrayerCard';
+import savedPrayerList from '../data/queries/getSavedPrayers';
+import unSavePrayer from '../data/mutations/unSavePrayer';
+
+const StyledView = styled(({ theme }) => ({
+  marginTop: theme.sizing.baseUnit * 1.5,
+  marginBottom: theme.sizing.baseUnit * 4,
+}))(View);
 
 const SavedPrayerList = () => (
   <ScrollView>
-    <PrayerCard />
-    <PrayerCard />
-    <PrayerCard />
-    <PrayerCard />
+    <Query query={savedPrayerList} fetchPolicy="cache-and-network">
+      {({ data: { savedPrayers = [] } = {} }) => (
+        <Mutation
+          mutation={unSavePrayer}
+          update={async (cache, { data: { unSavePrayer: returnedPrayer } }) => {
+            const currentSavedPrayers = cache.readQuery({
+              query: savedPrayerList,
+            });
+            const { id } = returnedPrayer;
+            const newPrayersList = currentSavedPrayers.savedPrayers.filter(
+              (prayer) => prayer.id !== id
+            );
+            await cache.writeQuery({
+              query: savedPrayerList,
+              data: { savedPrayers: newPrayersList },
+            });
+          }}
+        >
+          {(unSave) => (
+            <StyledView>
+              {savedPrayers
+                .map((prayer) => (
+                  <PrayerCard
+                    avatarSource={get(prayer, 'person.photo.uri')}
```

Comment:
usually a default value is useful, but that is up to you to decide if it would be useful in your component.

Reply:
After looking at it, it doesn't really need any defaults here. The `PrayerCard` is smart enough to know what to do without defaults or sets its own when necessary.